File: RadialMenu/RadialMenuJVM/src/main/java/RadialMenu/ContactMenu.java
package RadialMenu;

import com.spvessel.spacevil.ContextMenu;
import com.spvessel.spacevil.CoreWindow;
import com.spvessel.spacevil.MenuItem;
import com.spvessel.spacevil.PopUpMessage;

public class ContactMenu {
    ContextMenu _menu;
    MenuItem _call;
    MenuItem _remove;
    Contact _sender;
    RadialMenuItem _radialMenu = null;

    ContactMenu(CoreWindow window, RadialMenuItem radialMenu) {
        _radialMenu = radialMenu;
        _menu = new ContextMenu(window);
        _call = new MenuItem("Call");
        _remove = new MenuItem("Remove");

        _menu.setStyle(StyleFactory.getMenuStyle());
        _menu.addItems(_call, _remove);

        StyleFactory.getMenuItemStyle().setStyle(_call, _remove);

        initEvents();
    }

    public void linkContact(Contact contact) {
        contact.eventMouseClick.add((sender, args) -> {
            _menu.show(sender, args);
            _sender = contact;
        });
    }

    public void initEvents() {
        _remove.eventMouseClick.add((sender, args) -> {
            _sender.getParent().removeItem(_sender);
        });

        _call.eventMouseClick.add((sender, args) -> {
            _radialMenu.hide();
            PopUpMessage pop = new PopUpMessage("Calling to " + _sender.getName());
            pop.show(_menu.getHandler());
        });
    }
}

File: RadialMenu/RadialMenuNet/ContactMenu.cs
using System;
using SpaceVIL;

namespace RadialMenu
{
    public class ContactMenu
    {
        ContextMenu _menu;
        MenuItem _call;
        MenuItem _remove;
        Contact _sender;
        RadialMenuItem _radialMenu = null;

        public ContactMenu(CoreWindow window, RadialMenuItem radialMenu)
        {
            _radialMenu = radialMenu;
            _menu = new ContextMenu(window);
            _call = new MenuItem("Call");
            _remove = new MenuItem("Remove");

            _menu.SetStyle(StyleFactory.GetMenuStyle());
            _menu.AddItems(_call, _remove);

            StyleFactory.GetMenuItemStyle().SetStyle(_call, _remove);

            InitEvents();
        }

        public void LinkContact(Contact contact)
        {
            contact.EventMouseClick += (sender, args) =>
            {
                _menu.Show(sender, args);
                _sender = contact;
            };
        }

        public void InitEvents()
        {
            _remove.EventMouseClick += (sender, args) =>
            {
                _sender.GetParent().RemoveItem(_sender);
            };

            _call.EventMouseClick += (sender, args) =>
            {
                _radialMenu.Hide();
                PopUpMessage pop = new PopUpMessage("Calling to " + _sender.GetName());
                pop.Show(_menu.GetHandler());
            };
        }
    }
}